spot_ros2_control/hardware/include/spot_ros2_control/spot_hardware_interface.hpp

```diff
@@ -0,0 +1,131 @@
+// File modified. Modifications Copyright (c) 2024 Boston Dynamics AI Institute LLC.
+// All rights reserved.
+
+// --------------------------------------------------------------
+// Copyright 2020 ros2_control Development Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef SPOT_ROS2_CONTROL__SPOT_HPP_
+#define SPOT_ROS2_CONTROL__SPOT_HPP_
```

Comment:
spot_driver uses `#pragma once` do we want to maintain consistency?

Reply:
looking into this some more -- from first glance it seems like ros2 control code doesn't usually use `#pragma once`

Reply:
It's a c++ thing? Not a ros2 control thing?

Reply:
Yeah, I think just convention wise I am not seeing it used in the ros2 control source code / official examples, which is what I took the framework for this code from. But i would like for it to match with the spot driver

Reply:
I would recommend it, but if you are going to leave it as this type of header guard then you need to update it for the file name
`SPOT_ROS2_CONTROL__SPOT_HARDWARE_INTERFACE_HPP_`

That is another reason the `#pragma once` has become popular - you don't have to worry about file name changes.

Reply:
Added in latest commit